file/types.go

```diff
@@ -311,7 +311,7 @@ type FPlugin struct {
 
 // foo is a shadow type of Plugin.
 // It is used for custom marshalling of plugin.
-// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen=false
```

Comment:
Based on my read of https://github.com/kubernetes/gengo/blob/de9496dff47bef4ff8f45a56096bf50798cba2a5/examples/deepcopy-gen/generators/deepcopy.go#L177-L186, it seems we can delete this line and we will achieve the same effect.
The reason I would recommend deleting is because we don't enable `deepcopy-gen` at the package-level. If we did that, then setting this tag to `false` makes sense because we are doing exclusion for certain types.
In the current code, we are doing only inclusion-based selection for types that need a DeepCopy() so we should stick to that. Mixing inclusion and exclusion at the same time usually leads to more confusion later down the road.
Thoughts?

Reply:
No strong feelings from me either way. Pushed a new version that strips the tags.

Reply:
> Pushed a new version that strips the tags.

Doesn't seem like it. Can you check again?

Reply:
Hurray for `git commit --amend` and forgetting to add the file :facepalm: 

_Now_ it's properly updated.